cmangos/mangos-tbc
# SM Cath fix Mograine/Whitemane fight with CombatAI

## 🍰 Pullrequest

Basically the same as https://github.com/cmangos/mangos-tbc/pull/427 but with a crude CombatAI implementation
Additional issues fixed:
Whitemane now completes her run to the first waypoint before engaging
Whitemane resurrects Mograine earlier in the sleep phase

Issue that could not be fixed:
Whitemane doesn't salute to Mograine after resurrecting him. I tried, I failed.

### Proof

- None

### Issues

- None

### How2Test

- Create Character
- Set character level to 40
- `.additem 7146`
- `.tele scarletmonastery`
- Enter the left locked door to get to Scarlet Monastery Cathedral
- `.gm on` is probably easiest
- Move to the cathedral
- `.die` everything inside except for Scarlet Commander Mograine
- On yourself do:
- `.mod stam 100000`
- `.mod ap 5000`
- `.gm off` and engage Mograine
- After Mograine has died, Whitemane will appear and start casting Smite on you
- When Whitemane gets to 50% health, she'll cast Deep Sleep on everyone in range and resurrect Mograine
- Hit them until they both die

### Todo / Checklist

- [ ] Somehow get Salute working?

### Known Issues
- Playerbots don't quite play nicely with this script, as they seem to be able to continue hitting Mograine while he's faking death, thus re-aggroing him. This causes him to follow the playerbot around while lying on the ground and being untargetable. Don't know how to fix this :(
